db/transaction_log_impl.cc

```diff
@@ -234,21 +234,25 @@ void TransactionLogIteratorImpl::UpdateCurrentWriteBatch(const Slice& record) {
   // If the iterator has started, then confirm that we get continuous batches
   if (started_ && !IsBatchExpected(batch.get(), expectedSeq)) {
     // Seek to the batch having expected sequence number
+    bool fileChanged = false;
```

Comment:
naming convention is actually `file_changed`.
It's odd that TransactionLogIteratorImpl doesn't follow our naming convention as a whole, but I still hope the new variables follow it.